src/components/AudioPlayer.vue

```diff
@@ -95,13 +130,22 @@ export default {
   beforeDestroy() {
     this.audioPlayer.close();
   },
+  data() {
+    return {
+      // isSmallPlayer: true,
```

Comment:
It seems that this is not used anywhere, perhaps best to remove it?

Reply:
Done